sklearn/metrics/classification.py

```diff
@@ -680,16 +682,16 @@ def f1_score(y_true, y_pred, labels=None, pos_label=1, average='binary',
     Examples
     --------
     >>> from sklearn.metrics import f1_score
-    >>> y_true = [0, 1, 2, 0, 1, 2]
-    >>> y_pred = [0, 2, 1, 0, 0, 1]
+    >>> y_true = [0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 2, 2]
+    >>> y_pred = [0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 0, 1, 0]
     >>> f1_score(y_true, y_pred, average='macro')  # doctest: +ELLIPSIS
-    0.26...
+    0.50292397660818711
```

Comment:
I was hoping to signify the effect of data imbalance on the micro scores, i.e. with one class dominating the whole dataset by its size, then different averaging methods (macro, micro, weighted scores) tend to reveal different data characteristics.

In the previous example, for the macro score and weighted score, one really cannot tell the difference between them, since they both show 0.26. Same for the next example.

But I also notice that in the original classification.py file, all examples are short examples with less than 8 data points. Do you think it's necessary to include such examples with more points?